src/Schema/DataObject/DataObjectModel.php

```diff
@@ -382,6 +384,11 @@ public function getTypeName(): string
         return $this->getModelConfiguration()->getTypeName($class);
     }
 
+    public function getTypeDescription(): ?string
+    {
+        return $this->dataObject->classDescription() ?? null;
```

Comment:
```suggestion
        return $this->dataObject->i18nClassDescription() ?? null;
```
Unless there's a good reason to not use i18n here?

Reply:
I would argue that the description in the GraphQL schema is more like developer doc than CMS user doc. We don't normally localise the dev doc.

We also don't provide a way to localise the description when manually defining types or queries.

```yml
types:
  Monkey:
    description: "This description will only be available in English and can't be translated."
    fields:
      name:
        type: String
      age:
        type: Int
      species:
        type: String
```

It's worth pointing out that the description is set when you build your schema. The description will then be stored in the `.graphql-generated` folder. So if Friedrich has his CMS locale set to German and decides to rebuild the schema, everyone else will get their GraphQL schema description in German whether they like or not.